core/static/js/ga.js

```diff
@@ -0,0 +1 @@
+// override this file in your theme to add your own google analytics code
```

Comment:
I think docs and examples are good, but I still wouldn't have an analytics-specific block.  If we have a place to put arbitrary JS, we shouldn't additionally need a place to put analytics JS.  I could see that getting out of hand pretty fast.  The analytics block, the jquery plugins block, the CDN block, ....